.github/workflows/dependencies.yaml

```
@@ -0,0 +1,34 @@
+name: Update dependencies
+on:
+  schedule:
+    - cron: '0 3 * * 6'  # Run every saturday at 3 am.
+jobs:
+  update_dependencies:
+    name: Update dependencies
```

We shouldn't run enable this on forks.
```suggestion
    name: Update dependencies
    if: github.repository_owner == 'galaxyproject'
```